experiments/tf_trainer/keras_gru_attention/run.ml_engine.sh

```diff
@@ -12,11 +12,12 @@ gsutil acl set private ${REMOTE_COMET_API_KEY_FILE} &&
 gcloud ml-engine jobs submit training tf_trainer_${MODEL_NAME}_${USER}_${DATETIME} \
```

Comment:
Similarly a one line comment at the top stating that this dosn't do hyperparameter tuning and to use the other file instead.

Reply:
done

Reply:
done